inference-engine/src/vpu/myriad_plugin/myriad_plugin.cpp

```diff
@@ -32,6 +44,31 @@ using namespace InferenceEngine::PluginConfigParams;
 using namespace InferenceEngine::VPUConfigParams;
 using namespace vpu::MyriadPlugin;
 
+InferenceEngine::ICNNNetwork::Ptr Engine::CloneAndTransformNetwork(const InferenceEngine::ICNNNetwork& network) const {
+    auto clonedNetwork = cloneNetwork(network);
+
+    if (auto nGraphFunc = clonedNetwork->getFunction()) {
+            ngraph::op::GenericIE::DisableReshape noReshape(nGraphFunc);
+
+            ngraph::pass::Manager manager;
+
+            manager.register_pass<ngraph::pass::CommonOptimizations>();
+            manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
+            manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
+            manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
+
+            manager.run_passes(nGraphFunc);
+
+            ngraph::pass::Manager ti_manager;
+            ti_manager.register_pass<ngraph::pass::ApplyTransformationsToTIBody>(manager);
+            ti_manager.run_passes(nGraphFunc);
+
+            clonedNetwork = InferenceEngine::details::convertFunctionToICNNNetwork(nGraphFunc, *clonedNetwork);
```

Comment:
Is it copy-pasta from `frontend.cpp`? Can we reuse that code to not duplicate?

Reply:
Done